janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLKeyColumnValueStore.java

```diff
@@ -450,7 +473,7 @@ public CQLPagingIterator(final int pageSize, Supplier<Statement> statementSuppli
 
         @Override
         public boolean hasNext() {
-            return !currentResultSet.isExhausted();
+            return !currentResultSet.isFullyFetched();
```

Comment:
@porunov Looks like `isExhausted()` was removed in driver 4.x but it wasn't replaced by `isFullyFetched()` as that also existed in version 3.x. From looking at the implementation of `currentResultSet.one()` I assume that here we should be writing:
```java
            return currentResultSet.iterator().hasNext();
```
That seems to get a few more tests passing.